app/src/main/java/hr/dice/filipbionda/tmdbpractice/ui/theme/Color.kt

```diff
@@ -8,4 +8,10 @@ val white = Color.White
 val transparent = Color.Transparent
 val black_70 = Color(0xB20C0404)
 val lightPurple = Color(0xFF8000FF)
-val darkPurple = Color(0xFF540BA1)
+val purple_54 = Color(0xFF540BA1)
+val welcomeScreenPrimaryTextColor = Color.White
```

Comment:
name it white

Reply:
I deleted it because white already exists at beginning of the file